pkg/controllers/inventory/inventory_test.go

```diff
@@ -0,0 +1,93 @@
+package inventory
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+
+	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
+)
+
+var (
+	testVpcID01    = "testVpcID01"
+	testVpcName01  = "testVpcName01"
+	testVpcID02    = "testVpcID02"
+	testVpcName02  = "testVpcName02"
+	namespace      = "testNS"
+	accountName    = "account01"
+	namespacedName = types.NamespacedName{Namespace: namespace, Name: accountName}
+	region         = "xyz"
+	vpcCacheKey1   = fmt.Sprintf("%s/%s-%s", namespace, accountName, testVpcID01)
+	vpcCacheKey2   = fmt.Sprintf("%s/%s-%s", namespace, accountName, testVpcID02)
+	cloudInventory *Inventory
+)
+
+var _ = Describe("Validate Vpc Cache", func() {
+	BeforeEach(func() {
+		cloudInventory = InitInventory()
+	})
+
+	It("Build vpc cache", func() {
```

Comment:
Stimulate negative cases too to boost the coverage.. Like failed to add in the indexer

Reply:
I have tried it, had issues with it, we can discuss it.

Reply:
@Anandkumar26 can you help here

Reply:
Shared the patch with @archanapholla , will ask her to update the PR along with review comments.

Reply:
@Anandkumar26 @reachjainrahul even though we cannot hit the error cases through our UT, we need to retain the code, otherwise linter throws errors. Hence, I am keeping my code as is even though coverage drops a bit because of dead code.